packages/edit-site/src/components/dataviews/view-list.js

```diff
@@ -265,6 +276,38 @@ function ViewList( {
 		onGlobalFilterChange: ( value ) => {
 			onChangeView( { ...view, search: value, page: 0 } );
 		},
+		onColumnFiltersChange: ( filteringUpdater ) => {
+			onChangeView( ( currentView ) => {
+				const filters =
+					typeof filteringUpdater === 'function'
+						? filteringUpdater(
+								currentView.filters
+									? [
+											...Object.entries(
+												currentView.filters
+											).map( ( [ id, value ] ) => ( {
+												id,
+												value,
+											} ) ),
+									  ]
+									: []
+						  )
+						: filteringUpdater;
+
+				return {
+					...currentView,
+					filters: {
+						...filters.reduce(
+							( accumulator, { id, value } ) => ( {
+								...accumulator,
+								[ id ]: value,
```

Comment:
I think we want the "filter" for a given field to hold more than just the value, we probably need things like "operator" subkey later. Maybe it's fine to start that way but the format is probably going to change in the future.

The other thing is whether we should support nested parenthesis with logical operators, which most likely involves introducing a "type" key for a filter to indicate whether it's a "field" filter or a "group" of filters.

Reply:
This was a leftover from a previous experiment, your review landed when I was pushing changes for the new iteration :)